spec/serializers/api/v2/participant_serializer_spec.rb

```diff
@@ -0,0 +1,325 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Api
+  module V2
+    RSpec.describe ParticipantSerializer do
+      describe "serialization" do
+        let(:mentor_profile) { create(:mentor_participant_profile) }
+        let(:mentor) { mentor_profile.user }
+        let(:ect_profile) { create(:ect_participant_profile, mentor_profile: mentor_profile) }
+        let(:ect) { ect_profile.user }
+        let(:ect_cohort) { ect.early_career_teacher_profile.cohort }
+        let(:mentor_cohort) { mentor.mentor_profile.cohort }
+
+        context "when the participant is eligible for funding" do
+          before do
+            ECFParticipantEligibility.create!(participant_profile: ect_profile).eligible_status!
+            ECFParticipantEligibility.create!(participant_profile: mentor_profile).eligible_status!
+          end
+
+          it "outputs correctly formatted serialized Mentors" do
+            expected_json_string = "{\"data\":{\"id\":\"#{mentor.id}\",\"type\":\"participant\",\"attributes\":{\"email\":\"#{mentor.email}\",\"full_name\":\"#{mentor.full_name}\",\"mentor_id\":null,\"school_urn\":\"#{mentor.mentor_profile.school.urn}\",\"participant_type\":\"mentor\",\"cohort\":\"#{mentor_cohort.start_year}\",\"status\":\"active\",\"teacher_reference_number\":\"#{mentor.teacher_profile.trn}\",\"teacher_reference_number_validated\":true,\"eligible_for_funding\":true,\"pupil_premium_uplift\":false,\"sparsity_uplift\":false,\"training_status\":\"active\",\"schedule_identifier\":\"ecf-standard-september\",\"updated_at\":\"#{mentor.updated_at.rfc3339}\"}}}"
+            expect(ParticipantSerializer.new(mentor_profile).serializable_hash.to_json).to eq expected_json_string
```

Comment:
you don't really need to check against a JSON string. Just assert against the #serializable_hash. That will make things a tad easier to read

Reply:
Yeah i agree. It's copy/pasted so i'm not changing it as I've not touched it. next person to touch it can swap to a hash, i've done this for the tests I have modified